go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -34,6 +34,13 @@ const (
 	ctMarkOVN = "0x1"
 	// ctMarkHost is the conntrack mark value for host traffic
 	ctMarkHost = "0x2"
+	// ovnkubeITPMark is the fwmark used for host->ITP=local svc traffic. Note that the fwmark is not a part
+	// of the packet, but just stored by kernel in its memory to track/filter packet. Hence fwmark is lost as
+	// soon as packet exits the host.
+	ovnkubeITPMark = "0x1745ec" // constant itp(174)-service(5ec)
+	// ovnkubeSvcViaMgmPortRT is the number of the custom routing table used to steer host->service
+	// traffic packets into OVN via ovn-k8s-mp0. Currently only used for ITP=local traffic.
+	ovnkubeSvcViaMgmPortRT = "7"
```

Comment:
I couldn't find how or who created this routing table "7" is that precreated somehow ?

Reply:
yes so apparently if the custom table is just a number and not "named" it gets auto-created when we add the first route into it, thus we don't have to explicitly create it.